addons/common/functions/fnc_canDig.sqf

```diff
@@ -24,8 +24,10 @@ if ((getPosATL _unit) select 2 > 0.05 || // Walking on objects, such as building
 ) exitWith {false};
 
 private _surfaceClass = (surfaceType _posASL) select [1];
-private _surfaceType = getText (configFile >> "CfgSurfaces" >> _surfaceClass >> "soundEnviron");
-private _surfaceDust = getNumber (configFile >> "CfgSurfaces" >> _surfaceClass >> "dust");
+private _config = configFile >> "CfgSurfaces" >> _surfaceClass;
+private _surfaceType = getText (_config >> "soundEnviron");
+private _surfaceDust = getNumber (_config >> "dust");
+private _canDig = getNumber (_config >> "ACE_canDig") == 1;
 TRACE_2("Surface",_surfaceType,_surfaceDust);
 
-!(_surfaceType in DIG_SURFACE_BLACKLIST) && {(_surfaceDust >= 0.1) || {_surfaceType in DIG_SURFACE_WHITELIST}}
+_canDig || !{(_surfaceType in DIG_SURFACE_BLACKLIST) && {(_surfaceDust >= 0.1) || {_surfaceType in DIG_SURFACE_WHITELIST}}}
```

Comment:
This is what I meant:
```sqf
if (isNumber (_config >> "ACE_canDig")) then {
    getNumber (_config >> "ACE_canDig") // return
} else {
    !(_surfaceType in DIG_SURFACE_BLACKLIST) && {(_surfaceDust >= 0.1) || {_surfaceType in DIG_SURFACE_WHITELIST}} // return
};
```
Which makes the explicit setting of the entry overwrite whatever the script would estimate in case the entry is missing.

Reply:
This is the same logic that the description.ext and the mission.sqm operate under btw.